src/Request.php

```diff
@@ -314,7 +314,7 @@ public function getPost($name = null, $default = null)
      * Return the Cookie header, this is the same as calling $request->getHeaders()->get('Cookie');
      *
      * @convenience $request->getHeaders()->get('Cookie');
-     * @return Header\Cookie|bool
+     * @return Header\Cookie|false
```

Comment:
`false` is not a type: it's a value.

This change is invalid, sorry.

Reply:
@Ocramius Type can be explicit boolean value based on it http://docs.phpdoc.org/guides/types.html

Reply:
That's still not a type - even if phpdocumentor supports it, that is a mistake